README.md

```diff
@@ -200,13 +200,25 @@ summary_num_results: 10
 summary_prompt_name: vectara-summary-ext-v1.2.0
 ```
 
+#### Hybrid Search (optional)
+
+By default, vectara-answer utilizes hybrid search with lambda=0.1 for short queries (num_words<=2) and lambda=0 (pure neural search) otherwise, but other values can be defined here.
```

Comment:
Nit: We should strive for the active voice, for consistency in our UI, docs, repos, and so on. In this case I suggest rephrasing the last part of this sentence to:

> but you can define other values here.